firmware/src/board_defs.h

```diff
@@ -6,8 +6,8 @@
 #if defined BOARD_MAI_PICO
 
 #define I2C_PORT i2c1
-#define I2C_SDA 6
-#define I2C_SCL 7
+#define I2C_SDA 16
+#define I2C_SCL 17
```

Comment:
No, you can't publish your own board settings.

Reply:
Oh sorry I just accidentially committed this, I will revert this file later.